### PR TITLE
Add documentation

### PR DESCRIPTION
**Pull request by alesanter**

### CODE REVIEW

File: symplyphysics/laws/thermodynamics/heat_transfer/solution_with_zero_temperature_boundaries.py [hunk 1]
```diff
@@ -1,3 +1,25 @@
...
+
+.. _heat_transfer_zero_temperature_solution_coefficient_note:
+
+1. Values :math:`B_n` are found using the boundary condition :math:`f(x) = \sum_n T_n(x, 0)`
+   with the help of the Fourier method.
+2. The total solution :math:`T(x, t) = \sum_n T_n(x, t)`.
+
+**Conditions:**
+
+1. Position :math:`x \in [0, L]`.
```
Comment (blackyblack) [inline symplyphysics/laws/thermodynamics/heat_transfer/solution_with_zero_temperature_boundaries.py:20]: Add boundary condtions to conditions, i.e. `T_n(0, t) = 0`, `T_n(L, t) = 0`

File: symplyphysics/laws/thermodynamics/heat_transfer/solution_with_zero_temperature_boundaries.py [hunk 2]
```diff
@@ -1,3 +1,25 @@
...
+The heat equation coupled with a boundary condition can be solved to get a unique solution.
+In this boundary-value problem the heat transfer within a thin rod is observed, the temperature
+on both ends being zero. This restricts the number of functions :math:`f(x)` that can satisfy
+the boundary equation :math:`f(x) = T(x, 0)`.
+
+**Notes:**
+
+.. _heat_transfer_zero_temperature_solution_coefficient_note:
+
+1. Values :math:`B_n` are found using the boundary condition :math:`f(x) = \sum_n T_n(x, 0)`
```
Comment (blackyblack) [inline symplyphysics/laws/thermodynamics/heat_transfer/solution_with_zero_temperature_boundaries.py:14]: I think we are using `#.` for lists.

File: symplyphysics/laws/thermodynamics/heat_transfer/solution_with_zero_temperature_boundaries.py [hunk 3]
```diff
@@ -1,3 +1,25 @@
+r"""
+Solution with zero temperature boundaries
+=========================================
+
+The heat equation coupled with a boundary condition can be solved to get a unique solution.
+In this boundary-value problem the heat transfer within a thin rod is observed, the temperature
+on both ends being zero. This restricts the number of functions :math:`f(x)` that can satisfy
+the boundary equation :math:`f(x) = T(x, 0)`.
```
Comment (blackyblack) [inline symplyphysics/laws/thermodynamics/heat_transfer/solution_with_zero_temperature_boundaries.py:8]: Mention, that `f(x)` represents initial spatial distribution of temperature.

File: symplyphysics/laws/thermodynamics/maxwell_boltzmann_statistics/statistical_weight_of_macrostate.py [hunk 4]
```diff
@@ -11,33 +21,44 @@
...
 
 law = Eq(
     statistical_weight,
-    factorial(SumIndexed(particle_count[global_index], global_index)) /
-    ProductIndexed(factorial(particle_count[global_index]), global_index),
+    factorial(SumIndexed(particle_count_in_state[global_index], global_index)) /
+    ProductIndexed(factorial(particle_count_in_state[global_index]), global_index),
 )
+r"""
+:code:`W = factorial(N) / Product(factorial(N_i), i) = factorial(Sum(N_i, i)) / Product(factorial(N_i), i)`
```
Comment (blackyblack) [inline symplyphysics/laws/thermodynamics/maxwell_boltzmann_statistics/statistical_weight_of_macrostate.py:49]:
```suggestion
:code:`W = factorial(Sum(N_i, i)) / Product(factorial(N_i), i)`
```

File: symplyphysics/laws/thermodynamics/maxwell_boltzmann_statistics/statistical_weight_of_macrostate.py [hunk 5]
```diff
@@ -11,33 +21,44 @@
...
+
+Latex:
+    .. math::
+        W = \frac{N}{\prod_i (N_i!)} = \frac{(\sum_i N_i)!}{\prod_i (N_i!)}
```
Comment (blackyblack) [inline symplyphysics/laws/thermodynamics/maxwell_boltzmann_statistics/statistical_weight_of_macrostate.py:53]:
```suggestion
        W = \frac{(\sum_i N_i)!}{\prod_i (N_i!)}
```

Comment (blackyblack) [reply symplyphysics/laws/thermodynamics/maxwell_boltzmann_statistics/statistical_weight_of_macrostate.py:53]: You can mention alternative formula in notes.

File: symplyphysics/laws/waves/blackbody_radiation/spectral_energy_density_at_all_frequencies.py [hunk 6]
```diff
@@ -1,41 +1,75 @@
...
+Symbol:
+    :code:`u_nu`
+
+Latex:
+    :math:`u_\nu`
+"""
+
 radiation_frequency = Symbol("radiation_frequency", units.frequency)
+r"""
+(Linear) frequency of the radiation.
```
Comment (blackyblack) [inline symplyphysics/laws/waves/blackbody_radiation/spectral_energy_density_at_all_frequencies.py:46]:
```suggestion
Frequency (linear) of the radiation.
```

File: symplyphysics/laws/waves/blackbody_radiation/spectral_energy_density_in_high_frequency_limit.py [hunk 7]
```diff
@@ -13,30 +32,47 @@
...
-## - `h * nu >> k * T`, i.e. the photon energy is much greater than the thermal energy.
+Latex:
+    :math:`u_\nu`
+"""
 
-spectral_energy_density = Symbol("spectral_energy_density",
-    units.energy / (units.volume * units.frequency))
 radiation_frequency = Symbol("radiation_frequency", units.frequency)
+r"""
+(Linear) frequency of the radiation.
```
Comment (blackyblack) [inline symplyphysics/laws/waves/blackbody_radiation/spectral_energy_density_in_high_frequency_limit.py:49]:
```suggestion
Frequency (linear) of the radiation.
```

File: symplyphysics/laws/waves/blackbody_radiation/spectral_energy_density_in_low_frequency_limit.py [hunk 8]
```diff
@@ -13,33 +34,48 @@
...
-## - `h * nu << k * T`, i.e. the photon energy is much smaller than the thermal energy.
+Latex:
+    :math:`u_\nu`
+"""
 
-spectral_energy_density = Symbol("spectral_energy_density",
-    units.energy / (units.volume * units.frequency))
 radiation_frequency = Symbol("radiation_frequency", units.frequency)
+r"""
+(Linear) frequency of the radiation.
```
Comment (blackyblack) [inline symplyphysics/laws/waves/blackbody_radiation/spectral_energy_density_in_low_frequency_limit.py:51]:
```suggestion
Frequency (linear) of the radiation.
```

File: symplyphysics/laws/waves/relativistic/__init__.py [hunk 9]
```diff
@@ -0,0 +1,6 @@
+"""
+**Waves (relativistic)**
```
Comment (blackyblack) [inline symplyphysics/laws/waves/relativistic/__init__.py:2]:
```suggestion
**Waves (Relativistic)**
```

File: symplyphysics/laws/waves/wavelength_of_standing_wave_in_string_with_fixed_ends.py [hunk 10]
```diff
@@ -1,9 +1,17 @@
+"""
+Wavelength of standing wave in string with fixed ends
+=====================================================
+
+If a standing wave occurs in a string with fixed ends, the value of the displacement function
+must be zero on both ends of the string. To satisfy this boundary condition, only an integer
+number of half-waves can fit into the total length of the string.
```
Comment (blackyblack) [inline symplyphysics/laws/waves/wavelength_of_standing_wave_in_string_with_fixed_ends.py:7]: Add boundary condition to conditions.

File: symplyphysics/laws/waves/blackbody_radiation/spectral_energy_density_in_high_frequency_limit.py [hunk 11]
```diff
@@ -13,30 +32,47 @@
...
+"""
 
 law = Eq(spectral_energy_density, (8 * pi * planck * radiation_frequency**3 / speed_of_light**3) *
     exp(-1 * planck * radiation_frequency / (boltzmann_constant * equilibrium_temperature)))
+r"""
+:code:`u_nu = (8 * pi * h * nu^3 / c^3) * exp(-1 * h * nu / (k_B * T))`
+
+Latex:
+    .. math::
+        u_nu = \frac{8 \pi h \nu^3}{c^3} \exp \left( - \frac{h \nu}{k_B T} \right)
```
Comment (blackyblack) [inline symplyphysics/laws/waves/blackbody_radiation/spectral_energy_density_in_high_frequency_limit.py:74]:
```suggestion
        u_\nu = \frac{8 \pi h \nu^3}{c^3} \exp \left( - \frac{h \nu}{k_B T} \right)
```

File: symplyphysics/laws/electricity/chemistry/__init__.py [hunk 12]
```diff
@@ -0,0 +1,6 @@
+"""
+**Chemistry (electromagnetism)**
+================================
+
+Collection of laws related to electromagnetical description of chemical laws.
```
[the remark this answers is not in the document]
Comment (alesanter) [reply symplyphysics/laws/electricity/chemistry/__init__.py:5]: I guess you're right

**Review by blackyblack — APPROVED** [review]

**Pull request merged**